nav2_amcl/include/nav2_amcl/amcl_node.hpp

```diff
@@ -57,6 +57,9 @@ namespace nav2_amcl
  */
 class AmclNode : public nav2_util::LifecycleNode
 {
+  template<class M>
+  using Subscriber = message_filters::Subscriber<M, rclcpp_lifecycle::LifecycleNode>;
```

Comment:
Lets use this in line instead of `using` since its only used 1 time and I don't want people to glance over that code thinking its just a normal subscription